<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setmenu</class>
 <widget class="QWidget" name="Setmenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>0</y>
      <width>250</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>设置选项</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="but_defend">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>190</y>
      <width>200</width>
      <height>70</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>客户设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_debug">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>290</y>
      <width>200</width>
      <height>70</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>维护设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_user">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>90</y>
      <width>200</width>
      <height>70</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>监控设置</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>but_user</tabstop>
  <tabstop>but_defend</tabstop>
  <tabstop>but_debug</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
